comp/api/api/apiimpl/internal/config/endpoint.go

```diff
@@ -94,13 +99,28 @@ func (c *configEndpoint) getConfigValueAsJSON(r *http.Request) ([]byte, int, err
 	vars := gorilla.Vars(r)
 	path := vars["path"]
 
-	if _, ok := c.authorizedConfigPaths[path]; !ok {
+	authorized := false
+	if _, ok := c.authorizedConfigPaths[path]; ok {
+		authorized = true
+	} else {
+		// check to see if we have a prefix rule path (trailing period) which matches as the prefix
+		// for the given path
+		for path := range c.authorizedConfigPaths {
+			if strings.HasSuffix(path, ".") && strings.HasPrefix(path, path) {
```

Comment:
You are shadowing the `path` variable, `strings.HasPrefix(path, path)` is always true, and `strings.HasSuffix(path, ".")` is true due to `ha.`, so now any config can be queried :smile:

```
❯ curl -L -k -H "authorization: Bearer $(cat dev/dist/auth_token)" "https://localhost:5004/config/v1/proxy"
{"http":"","https":"","no_proxy":["169.254.169.254","100.100.100.200"]}%
```

Reply:
Rut roh. 😰 

Will fix.

Reply:
Fixed in ac35e82.